pygmt/figure.py

```diff
@@ -59,7 +59,8 @@ class Figure:
     Examples
     --------
 
-    >>> fig = Figure()
+    >>> import pygmt
+    >>> fig = pygmt.Figure()
     >>> fig.basemap(region=[0, 360, -90, 90], projection="W7i", frame=True)
```

Comment:
Please change `7i` to something like `15c`, because we recommend using SI units in both GMT and PyGMT.

Reply:
Done in https://github.com/GenericMappingTools/pygmt/pull/2105/commits/e84f09b9a4c7daadafa66098e3fb0b238c411c71.
Should I also update the example in the Contributors Guide: https://www.pygmt.org/dev/contributing.html#using-mpl-image-compare?

Reply:
> Should I also update the [example in the Contributors Guide](https://www.pygmt.org/dev/contributing.html#using-mpl-image-compare) and the [Legend example](https://www.pygmt.org/dev/gallery/embellishments/legend.html#sphx-glr-gallery-embellishments-legend-py)? 

Yes.

> In the [Plotting vectors tutorial](https://www.pygmt.org/dev/tutorials/advanced/vectors.html?highlight=X5i#plot-cartesian-vectors) it looks like there was a specific intention behind using inches.

Not necessary for this example.